DependencyInjection/SwarrotExtension.php

```diff
@@ -29,14 +29,10 @@ public function load(array $configs, ContainerBuilder $container)
             throw new \InvalidArgumentException('Unsupported provider');
         }
 
-        $definition = $container->getDefinition($id);
-
         foreach ($config['connections'] as $name => $connectionConfig) {
-            $definition->addMethodCall('addConnection', array(
-                $name,
-                $connectionConfig
-            ));
+            $connections[$name] = $connectionConfig;
         }
+        $container->setParameter('swarrot.connections', $connections);
```

Comment:
I closed this PR without merging it.
In fact, I needed this connections list to be able to reuse them in another bundle that's why I wished I had this parameter. :)

PR #67 solved my problem as it already expose a parameter containing the connection list.